pkgs/by-name/k2/k2pdfopt/package.nix

```diff
@@ -25,8 +25,9 @@
   # Tesseract support is currently broken
   # See: https://github.com/NixOS/nixpkgs/issues/368349
   enableTesseract ? false,
-  leptonica,
   tesseract5,
+  enableLeptonica ? false,
```

Comment:
Does this mean `leptonica` should actually stay disabled instead, or is this a typo?

I'm asking, because I don't know anything about those dependencies.

Also the commit message still says that we're re-enabling it.